include/swift/Option/Options.td

```diff
@@ -805,6 +805,10 @@ def experimental_package_interface_load:
   Flags<[FrontendOption, HelpHidden]>,
   HelpText<"Enables loading a package interface if in the same package specified with package-name">;
 
+def experimental_serialize_debug_info:
+  Flag<["-"], "experimental-serialize-debug-info">,
+  Flags<[FrontendOption, HelpHidden]>,
+  HelpText<"Enables seriailzation/deserialization of debug scopes">;
```

Comment:
This should become on  by default eventually, right?

Reply:
Yes it should be. I am not sure if we want to turn it on right now. @kubamracek Do you have a suggestion here? Thanks

Reply:
Yeah, I think this is a staging flag, correct? Sounds like a good approach to me.